README.md

```diff
@@ -168,6 +168,85 @@ batch.patchItemOperation(testDoc.getId(), cosmosPatchOperations);
 TransactionalBatchResponse batchResponse = container.executeTransactionalBatch(batch).block();
 ~~~~
 
+### NodeJS:
+
+The npm package can be found at [NPM package: com.azure » @azure/cosmos » 3.14.1 (npmjs.com)](https://www.npmjs.com/package/@azure/cosmos/v/3.14.1)
+
+
+#### Patching an item with a single patch operation
+
+```
+const patchSource = testItem;
```

Comment:
this line seems redundant to me; I would delete this line and replace all instances of `patchSource` with `testItem` or `testDoc` the way the samples above do